third_party/CMakeLists.txt

```diff
@@ -93,3 +93,45 @@ if(VELOX_ENABLE_ARROW)
                                            ${ARROW_LIBDIR}/libparquet.a)
 
 endif()
+
+if(VELOX_ENABLE_QPL)
+  message(STATUS "Building QPL from source")
+  set(QPL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/qpl_ep/install")
+  set(QPL_STATIC_LIB_NAME
+      ${CMAKE_STATIC_LIBRARY_PREFIX}qpl${CMAKE_STATIC_LIBRARY_SUFFIX})
+  set(QPL_STATIC_LIB "${QPL_PREFIX}/lib/${QPL_STATIC_LIB_NAME}")
+
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ldl -laccel-config -L/usr/lib64")
```

Comment:
We use CXX_FLAGS in velox  but it's a deprecated practice (it changes the flags for ALL targets) and we want to move away from it so please use the target based functions cmake provides to apply these flags only to the target that needs them. (or in this case use a different variable name to apply them only to the ep as this looks to be your intention?)

Reply:
This was not addressed

Reply:
I removed this line from ```CMake/resolve_dependency_modules/inteliaa.cmake```, ```third_party/CMakeLists.txt``` is deprecated and ```inteliaa.cmake``` is an substitute.